src/components/Infopic/Infopic.tsx

```diff
@@ -36,32 +36,32 @@ const TextComponent = ({
   buttonUrl,
 }: Omit<InfopicProps, "sectionIndex" | "image" | "alt">) => {
   return (
-    <div className="lg:max-w-lg">
-      <div className="mt-4">
-        <div>
-          <p className="text-subtitle pb-4 uppercase tracking-widest">
-            {subtitle}
-          </p>
-        </div>
-        <h1 className="text-secondary text-5xl font-semibold pb-4">{title}</h1>
-      </div>
-
-      <section aria-labelledby="information-heading" className="mt-4">
-        <div className="mt-4 space-y-6">
+    <div className="flex flex-col gap-4 lg:px-8">
+      {subtitle && (
+        <p className="text-subtitle uppercase tracking-widest">{subtitle}</p>
+      )}
+      {title && (
+        <h1 className="text-secondary text-5xl font-semibold">{title}</h1>
+      )}
+      {title && (
```

Comment:
question(blocking): should this be description instead?

Reply:
[5edfc06](https://github.com/isomerpages/isomer-components/pull/5/commits/5edfc0634cdb7ca8e9d1d4c25f89a9a489c14be6)